docs/reference/release-notes/highlights.asciidoc

```diff
@@ -38,4 +71,13 @@ succession. Rather than type the same event criteria multiple times, you can use
 a `with runs` statement to declare the criteria once and run it successively.
 For more details, check out the
 {ref}/eql-syntax.html#eql-with-runs-statement[EQL syntax documentation].
+
+
+=== Field capabilities: Results gathered by node rather than index
+
+Previously, field caps were gathered by sending a request per index to the
+appropriate nodes in the cluster resulting in high overhead when many indices
+were targeted. Now requests that target indices on a single node are grouped
+together resulting in no more than a single request per node.
```

Comment:
I'd consider combining this with the "can-match" section or moving it side-by-side. The benefits and takeaways are largely the same.

Reply:
Thanks, @jrodewig. I've made those changes.